Functions/xASL_Iteration.m

```diff
@@ -16,20 +16,27 @@
 % OUTPUT:
 %   bAborted        - Report if the run was aborted
 %   xOut            - x-struct on the output
+%
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % DESCRIPTION: Parses the settings and runs the DatabaseLoop sub-function.
+%
 % EXAMPLE:  [~,x] = xASL_Iteration(x,'xASL_module_ASL');
-% -----------------------------------------------------------------------------------------------------------------------------------------------------
 %
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
 % __________________________________
-% Copyright (C) 2015-2019 ExploreASL
+% Copyright (C) 2015-2021 ExploreASL
 
     % General settings
     dbSettings.x                     = x;
     dbSettings.x.settings.RERUN      = false;
     dbSettings.x.settings.MUTEXID    = moduleName;
     dbSettings.x.dir.LockDir         = ['<ROOT>/lock/' moduleName];
     
+    % Lock dir specifics for Import
+    if strcmp(moduleName,'xASL_module_Import')
```

Comment:
Nice, but this shouldn't be specific to the import module right, this is the same location as the processing modules?

Reply:
Yes and no. During the import the `<ROOT>` string is our BIDS dataset root. During processing, `<ROOT>` historically was the root of the ExploreASL legacy data. In theory we could change this for the processing, but that would require more extensive testing and right now everything seems to be stable.